site/index.html

```diff
@@ -2,7 +2,8 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <title>Home</title>
+    <meta name="robots" content="noindex" />

```

Comment:
Is it enough ? or we need `robots.txt` at the root ?

Reply:
This is the only HTML served from the domain so any page would be crawled with that attribute https://developers.google.com/search/docs/advanced/robots/robots_meta_tag

Reply:
ok.